studio/src/rte/rte-link-editor.js

```diff
@@ -170,6 +170,14 @@ export class RteLinkEditor extends LitElement {
                     variant="accent"
                     >Accent</sp-button
                 >
+                <sp-button
```

Comment:
what is the difference between primary or accent? Cannot you use accent ?

Reply:
Primary is black by default whereas accent is blue.